share/spice/political_polls/political_polls.js

```diff
@@ -0,0 +1,58 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_political_polls = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('political_polls');
+        }
+        
+        var results = api_result;
+
+        if (!results) {
```

Comment:
Looks like this is redundant because you already checked the truthiness of `api_result`